lr_face/data.py

```diff
@@ -51,13 +56,16 @@ def get_image(self,
 
         :param resolution: Optional[Tuple[int, int]]
         :param normalize: bool
+        :param augmenter: Optional[Augmenter]
         :return: np.ndarray
         """
         res = cv2.imread(self.path)
         if res is None:
             raise ValueError(f'Reading {self.path} resulted in None')
         if res.shape[-1] != 3:
             raise ValueError(f'Expected 3 channels, got {res.shape[-1]}')
+        if augmenter:
```

Comment:
is dit de plek? dit is toch geen eigenschap van het plaatje?
also, is dit van dit PR?